bin/import-test-content.php

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/php
+<?php
+
+namespace WPOrg_Learn\Bin\ImportTestContent;
+
+/**
+ * CLI script for generating local test content, fetched from the live learn.wordpress.org site.
+ *
+ * This needs to be run in a wp-env, for example:
+ *
+ * yarn run wp-env run cli "php bin/import-test-content.php"
+ */
+
+// This script should only be called in a CLI environment.
+if ( 'cli' != php_sapi_name() ) {
+    die();
+}
+
+
+$opts = getopt( '', array( 'post:', 'url:', 'abspath:', 'age:' ) );
+
+require dirname( dirname( __FILE__ ) ) . '/wp-load.php';
+
+// Refuse to run on a production site.
+$run_on_hosts = [
+	'localhost',
+	'127.0.0.1',
+];
+if ( !in_array( parse_url( get_site_url(), PHP_URL_HOST ), $run_on_hosts ) ) {
+	die( "Not safe to run on " . get_site_url() );
+}
+
+function sanitize_meta_input( $meta ) {
+	$meta = (array( $meta ) );
+	foreach ( $meta as $k => $v ) {
+		if ( is_array( $v ) ) {
+			$meta[ $k ] = implode( ',', $v );
+		}
+	}
+
+	return $meta;
+}
+
+function import_rest_to_posts( $rest_url ) {
+	$response = wp_remote_get( $rest_url );
+	if ( is_wp_error( $response ) ) {
+		die( $response->get_error_message() );
+	}
```

Comment:
```suggestion
	$response = wp_remote_get( $rest_url );
	$status_code = wp_remote_retrieve_response_code( $response );

	if ( is_wp_error( $response ) ) {
		die( $response->get_error_message() );
	} elseif ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
		die( "HTTP Error $status_code \n" );
	}
```

I applied the branch to my sandbox, and the API endpoints are working. I get a `403 Forbidden` error when trying running this locally, though. I think it's just the same problem that we've been seeing in other situations, caused by recent changes to the sandboxes.